soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3037,6 +3044,27 @@ void GenerateRandomizerImgui(std::string seed = "") {
     cvarSettings[RSK_MIX_INTERIOR_ENTRANCES] = CVarGetInteger("gRandomizeMixInteriors", RO_GENERIC_OFF);
     cvarSettings[RSK_MIX_GROTTO_ENTRANCES] = CVarGetInteger("gRandomizeMixGrottos", RO_GENERIC_OFF);
     cvarSettings[RSK_DECOUPLED_ENTRANCES] = CVarGetInteger("gRandomizeDecoupleEntrances", RO_GENERIC_OFF);
+    cvarSettings[RSK_ALLOW_DECOUPLED_ENTRANCES] = CVarGetInteger("gAllowDecoupledEntrances", RO_GENERIC_OFF);
+
+    RandomSettingsOptions randomOptions = { 
+        cvarSettings[RSK_RANDOMIZE_WORLD_SETTINGS] == RO_GENERIC_ON,
+        cvarSettings[RSK_RANDOMIZE_ITEM_SETTINGS] == RO_GENERIC_ON,
+        cvarSettings[RSK_RANDOMIZE_GAMEPLAY_SETTINGS] == RO_GENERIC_ON,
+        cvarSettings[RSK_RANDOMIZE_STARTING_INVENTORY_SETTINGS] == RO_GENERIC_ON,
+    };
+
+    srand(time(NULL));
```

Comment:
C's rand function should not be used in a C++ file.  https://github.com/HarbourMasters/Shipwright/blob/develop/soh/soh/Enhancements/randomizer/3drando/random.cpp has all the functions you'll need to get random ints.

Reply:
This is fixed in latest commits.